src/ops/ops.go

```diff
@@ -437,3 +440,24 @@ func (o *ops) ReloadHostFile(filepath string) error {
 	}
 	return nil
 }
+
+func (o *ops) CreateOpenshiftSshManifest(filePath, tmpl string) error {
+	o.log.Info("Create an openshift manifets for SSH public key")
+	sshPublicKey, err := ioutil.ReadFile("/root/.ssh/id_rsa.pub")
```

Comment:
consts?

Reply:
This is the only place it is used, I think it is more  clear this way.

Reply:
done